web/src/components/common/GreenButton.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import styled from 'styled-components';
+
+import Button from './Button';
+
+/**
+ * Button with green background, white uppercase fonts and rounder corners.
+ */
+const GreenButton = styled(Button)`
+  background: var(--green);
+`;
```

Comment:
Hmmmm for components like buttons I prefer that it's not a separate component on its own. Instead, you should make `Button` a general component and allow it to take in optional props of `color` where users can specify the color they want themselves. 

So you can do something like
```
<Button color="green">...</Button>
```
or allow user to fully customise the color they want 
```
<Button color="var(--green)">...</Button>
```
or you can do it the bootstrap-y way of accepting `"primary" | "danger"  | ...`  as the props etc

Reply:
@hzxcaryn Oh yep! Was planning to do that also hahah